src/main/flight/pid.h

```diff
@@ -152,6 +152,8 @@ typedef struct pidProfile_s {
     uint8_t  antiGravityMode;             // type of anti gravity method
     uint16_t itermThrottleThreshold;        // max allowed throttle delta before iterm accelerated in ms
     uint16_t itermAcceleratorGain;          // Iterm Accelerator Gain when itermThrottlethreshold is hit
+    uint8_t itermAcceleratorThrottleChangeMaxLimit; // Maximum value for the throttle change for Anti Gravity
+    uint8_t itermAcceleratorPboostMultiplier;       // P boost multiplier for Anti Gravity (100 = 1.0)
```

Comment:
These are quite long names, now that anti-gravity boosts both p and iterm it would make sense to just remove the itermAccelerator from the name and rename them to antiGravity.

Reply:
any renaming probably should go to another PR to make sure PR is easy to follow.